NEWS

```diff
@@ -1,6 +1,22 @@
 Ticket reference tags refer to tickets in the MongoDB JIRA tracking system:
 https://jira.mongodb.org
 
+WiredTiger release 10.0.0, 2020-11-25
+------------------------------------
+
+See the upgrading documentation for details of API and behavior changes.
+
+Significant changes:
+* WT-4833	Drop support for Python2
+* WT-6410	Remove WT_SESSION.rebalance
+* WT-6674	Remove Async API code and documentation
+* WT-6675	Remove WiredTiger Java language API and documentation
+* WT-6678	Remove Huffman Encoding support for Keys
+* WT-6713	Remove transaction support from custom data sources
```

Comment:
@lukech Thanks for summarizing the behaviour changes. There is one more change that went in yesterday is `default transaction isolation` is changed from `read-committed` to `snapshot` as part of WT-6710. If we are going to include those commits also in the release, we need to update the NEWS and upgrade sections. Otherwise, it looks good.

Reply:
Thanks @kommiharibabu. Added the default isolation change into the docs using 93c1a41.